apps/src/javalab/JavalabFileHelper.js

```diff
@@ -42,6 +102,19 @@ export const fileMetadataForEditor = (sources, isEditingStartSources) => {
   };
 };
 
+export const updateAllSourceFileOrders = (
+  sources,
+  fileMetadata,
+  orderedTabKeys
+) => {
+  for (let i = 0; i < orderedTabKeys.length; i++) {
```

Comment:
It's ok to occasionally use `i` for an array index, but it's generally better to use a variable name that describes the purpose of `i`. 

Some context: we used to use a lot of abbreviations in code to decrease the number of keystrokes and the size of programs. But now that we have auto-complete and code minification, it's generally best practice to use full words to make the code more readable for the next developer. `i` is one of those abbreviations that has stuck around, but use it with care. Seeing it used in multiple places in this file starts to point to "code smell"

Reply:
Thanks Jessie! This definitely makes sense.